pennylane/ops/qubit.py

```diff
@@ -480,13 +480,13 @@ class CRot(Operation):
 
 class QubitUnitary(Operation):
     r"""QubitUnitary(U, wires)
-    Apply an arbitrary unitary matrix
+    Apply an arbitrary fixed unitary matrix; QubitUnitary only accepts unitary matrices which do not depend on any parameters.
 
     **Details:**
 
-    * Number of wires: None (applied to the entire system)
+    * Number of wires: The operation can act on any number of wires.
     * Number of parameters: 1
-    * Gradient recipe: None (uses finite difference)
+    * Gradient recipe: None (the operation may not be differentiated)
```

Comment:
It might be clearer just to have `None`, thoughts? Mostly because saying 'Operation may not be differentiated' makes it sound like you can't use this operation in a circuit you want to differentiate.
```suggestion
    * Gradient recipe: None
```

Reply:
hmmm It is a possible interpretation. I will change this! Thanks!